outputs/costs_2020.csv

```diff
@@ -75,6 +93,24 @@ General liquid hydrocarbon storage (crude),lifetime,30.0,years,"Stelter and Nish
+Gravity-Water-Aboveground-bicharger,FOM,1.5,%/year,"Viswanathan_2022, p.76 (p.98) Sentence 1 in 4.7.2 Operating Costs","{'carrier': ['elec', 'gravitywa', 'elec'], 'technology_type': ['bicharger'], 'type': ['mechanical'], 'note': ['1.5 percent of capital cost']}"
+Gravity-Water-Aboveground-bicharger,efficiency,0.9,per unit,"Viswanathan_2022, p.77 (p.99) Table 4.37","{'carrier': ['elec', 'gravitywa', 'elec'], 'technology_type': ['bicharger'], 'type': ['mechanical'], 'note': ['AC-AC efficiency at transformer level ((0.785+0.84)/2)^0.5']}"
+Gravity-Water-Aboveground-bicharger,investment,365630.71,EUR/MW,"Viswanathan_2022, p.71 (p.94) text at the bottom speaks about 0% cost reduction for 2030 compared to 2021","{'carrier': ['elec', 'gravitywa', 'elec'], 'technology_type': ['bicharger'], 'type': ['mechanical'], 'note': ['Power Equipment']}"
+Gravity-Water-Aboveground-bicharger,lifetime,60.0,years,"Viswanathan_2022, p.77 (p.99) Table 4.37","{'carrier': ['elec', 'gravitywa', 'elec'], 'technology_type': ['bicharger'], 'type': ['mechanical'], 'note': ['NULL']}"
+Gravity-Water-Aboveground-store,investment,144713.47,EUR/MWh,"Viswanathan_2022, p.71 (p.94) text at the bottom speaks about 15% cost reduction for 2030 compared to 2021","{'carrier': ['gravitywa'], 'technology_type': ['store'], 'type': ['mechanical'], 'note': ['Gravitational Capital (SB+BOS)']}"
+Gravity-Water-Aboveground-store,lifetime,60.0,years,"Viswanathan_2022, p.77 (p.99) Table 4.37","{'carrier': ['gravitywa'], 'technology_type': ['store'], 'type': ['mechanical'], 'note': ['NULL']}"
+Gravity-Water-Underground-bicharger,FOM,1.5,%/year,"Viswanathan_2022, p.76 (p.98) Sentence 1 in 4.7.2 Operating Costs","{'carrier': ['elec', 'gravitywu', 'elec'], 'technology_type': ['bicharger'], 'type': ['mechanical'], 'note': ['1.5 percent of capital cost']}"
+Gravity-Water-Underground-bicharger,efficiency,0.9,per unit,"Viswanathan_2022, p.77 (p.99) Table 4.37","{'carrier': ['elec', 'gravitywu', 'elec'], 'technology_type': ['bicharger'], 'type': ['mechanical'], 'note': ['AC-AC efficiency at transformer level ((0.785+0.84)/2)^0.5']}"
+Gravity-Water-Underground-bicharger,investment,905158.96,EUR/MW,"Viswanathan_2022, p.71 (p.94) text at the bottom speaks about 0% cost reduction for 2030 compared to 2021","{'carrier': ['elec', 'gravitywu', 'elec'], 'technology_type': ['bicharger'], 'type': ['mechanical'], 'note': ['Power Equipment']}"
+Gravity-Water-Underground-bicharger,lifetime,60.0,years,"Viswanathan_2022, p.77 (p.99) Table 4.37","{'carrier': ['elec', 'gravitywu', 'elec'], 'technology_type': ['bicharger'], 'type': ['mechanical'], 'note': ['NULL']}"
+Gravity-Water-Underground-store,investment,113887.53,EUR/MWh,"Viswanathan_2022, p.71 (p.94) text at the bottom speaks about 15% cost reduction for 2030 compared to 2021","{'carrier': ['gravitywu'], 'technology_type': ['store'], 'type': ['mechanical'], 'note': ['Gravitational Capital (SB+BOS)']}"
```

Comment:
The dictionary at the end will probably cause problems because it also uses "," as separator?

Reply:
Yeah, I was wondering about that as well. 
I tested the csv import and it works smoothly.
Maybe because the cell was quoted?

Reply:
Here is the processing: https://github.com/pypsa-meets-earth/pypsa-earth/blob/322652f162cffc20250d397d75633e3c6548b42c/scripts/_helpers.py#L612

In this PR, you can already use PyPSA-Earth to optimize all technologies: https://github.com/pypsa-meets-earth/pypsa-earth/pull/567